projects/media-viewer/src/lib/toolbar/main-toolbar/right-pane/right-pane.component.html

```diff
@@ -1,4 +1,9 @@
 <div id="toolbarViewerRight">
+  <button *ngIf="toolbarButtons.showCommentSummary && toolbarButtons.showDrawButton"
```

Comment:
why do we need to check for drawButton?

Reply:
in order  to check if annotations are enabled - draw button is shown for both pdf and images when annotations are enabled - could you suggest a better way?